### PR TITLE
fix(genymotion-bin): fix invalid desktop icon file

### PR DESCRIPTION
**Pull request by Arrowsome**

### CODE REVIEW

File: packages/genymotion-bin/genymotion-bin.pacscript [hunk 1]
```diff
@@ -14,5 +14,5 @@ maintainer="Arrowsome <ramtintoosi@gmail.com>"
 
 package() {
   sudo chmod +x "./genymotion-${pkgver}-linux_x64.bin"
-  sudo "./genymotion-${pkgver}-linux_x64.bin" -y -d "${pkgdir}/opt"
+  sudo "./genymotion-${pkgver}-linux_x64.bin" -y -d "/opt"
```
Comment (Elsie19) [inline packages/genymotion-bin/genymotion-bin.pacscript:17]: Everything needs to be inside `${pkgdir}` for `package`.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/genymotion-bin/genymotion-bin.pacscript (hunk 1 above)
Comment (Arrowsome) [reply packages/genymotion-bin/genymotion-bin.pacscript:17]:
```
[Desktop Entry]
...
Icon=/usr/src/pacstall/genymotion-bin/opt/genymotion/icons/genymotion-logo.png
Exec=/usr/src/pacstall/genymotion-bin/opt/genymotion/genymotion
...
```

The desktop icon has invalid path in this case!

Comment (Elsie19) [reply packages/genymotion-bin/genymotion-bin.pacscript:17]: Then `sed` it out during `package()`, such as `sed -i 's|/usr/src/pacstall/genymotion-bin||g' path/to/desktop/file`!

File: packages/genymotion-bin/genymotion-bin.pacscript [hunk 2]
```diff
@@ -15,4 +15,5 @@ maintainer="Arrowsome <ramtintoosi@gmail.com>"
 package() {
   sudo chmod +x "./genymotion-${pkgver}-linux_x64.bin"
   sudo "./genymotion-${pkgver}-linux_x64.bin" -y -d "${pkgdir}/opt"
+  sudo sed -i "s|${pkgdir}||g" /usr/share/applications/genymobile-genymotion.desktop
```
Comment (Elsie19) [inline packages/genymotion-bin/genymotion-bin.pacscript:18]: this should have `${pkgdir}` before the path to the desktop.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/genymotion-bin/genymotion-bin.pacscript (hunk 2 above)
Comment (Arrowsome) [reply packages/genymotion-bin/genymotion-bin.pacscript:18]: In that case: 
```shell
sudo sed -i "s|${pkgdir}||g" "${pkgdir}/usr/share/applications/genymobile-genymotion.desktop"`
```
_sed: can't read /usr/src/pacstall/genymotion-bin/usr/share/applications/genymobile-genymotion.desktop: No such file or directory_

Comment (Elsie19) [reply packages/genymotion-bin/genymotion-bin.pacscript:18]: Why are you installing a file straight to root? Everything goes in `${pkgdir}`.

File: packages/genymotion-bin/genymotion-bin.pacscript [hunk 3]
```diff
@@ -5,14 +5,45 @@ pkgver="3.6.0"
...
 maintainer="Arrowsome <ramtintoosi@gmail.com>"
 
+_GENYMOTION_FILE="./genymotion-${pkgver}-linux_x64.bin"
+_GENYMOTION_INSTALLDIR="${pkgdir}/opt/genymotion"
+
+prepare() {
+  sudo rm -rf "/opt/${gives}"
+  sudo mkdir -p "${_GENYMOTION_INSTALLDIR}"
+
+  sudo chmod +x "${_GENYMOTION_FILE}"
```
Comment (Elsie19) [inline packages/genymotion-bin/genymotion-bin.pacscript:21]: Where is this being installed in/before `prepare`?

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/genymotion-bin/genymotion-bin.pacscript (hunk 3 above)
Comment (Arrowsome) [reply packages/genymotion-bin/genymotion-bin.pacscript:21]: hmm, what do you mean by "this"? I don't understand the question!

Comment (Elsie19) [reply packages/genymotion-bin/genymotion-bin.pacscript:21]: Oh sorry I was tripping yesterday. I don't know why I didn't read the entire script.

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**